sdk/exportentities/workflow.go

```diff
@@ -468,70 +447,71 @@ func (w Workflow) GetWorkflow() (*sdk.Workflow, error) {
 		wf.Groups = append(wf.Groups, perm)
 	}
 
-	wf.Sort()
+	wf.SortNode()
 
 	return wf, nil
 }
 
-func (e *NodeEntry) getNode(name string) (*sdk.WorkflowNode, error) {
-	node := &sdk.WorkflowNode{
-		Name:         name,
-		Ref:          name,
-		PipelineName: e.PipelineName,
+func (e *NodeEntry) getNode(name string, proj *sdk.Project, w *sdk.Workflow) (*sdk.Node, error) {
+	node := &sdk.Node{
+		Name:    name,
+		Ref:     name,
+		Context: &sdk.NodeContext{},
 	}
 
-	if e.ApplicationName != "" {
-		node.Context = new(sdk.WorkflowNodeContext)
-		node.Context.Application = &sdk.Application{
-			Name: e.ApplicationName,
+	if e.PipelineName != "" {
+		for _, v := range proj.Pipelines {
+			if v.Name == e.PipelineName {
+				node.Context.PipelineID = v.ID
+				break
+			}
+		}
+		node.Type = sdk.NodeTypePipeline
+	} else {
+		if e.OutgoingHookModelName == "" {
```

Comment:
if it has no pipeline and not outgoing hook model, it's a fork ?
the fork is kinda default type ?

Reply:
for now it can be a fork or a join. 
But join is compute after